rllib/agents/dreamer/dreamer_model.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
 from typing import Any, List, Tuple
+from ray.rllib.models.torch.modules.reshape import Reshape
```

Comment:
This line crashes. The previous dialog indicates that you seem to move it to ray.rllib.models.torch.misc .